Google.Api.Generator.Utils/Roslyn/RoslynBuilder.cs

```diff
@@ -233,6 +233,8 @@ public static PropertyDeclarationSyntax AutoProperty(Modifier modifiers, TypeSyn
         public static CodeFunc<ForStatementSyntax> For(LocalDeclarationStatementSyntax decl, ExpressionSyntax condition, ExpressionSyntax incr) => code =>
             ForStatement(decl.Declaration, SeparatedList<ExpressionSyntax>(), condition, SingletonSeparatedList(incr), Block(ToStatements(code)));
 
+        public static ExpressionSyntax ToDefaultExpression(object foo) => ToExpression(foo);
```

Comment:
Not sure what this new method is needed for? (Or what it does.) Let's remove it if we don't need it, but I'm fine if we do.

Reply:
👍